### PR TITLE
Reboot Mongo clusters unattended in Production

### PR DESCRIPTION
**Pull request by mattbostock**
Story: https://trello.com/c/WRgYW2Zi/14-write-script-to-check-if-mongo-api-mongo-machines-can-be-rebooted

Enable unattended server reboots for MongoDB secondaries in Production.

### CODE REVIEW

File: hieradata/class/licensify_mongo.yaml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 
-govuk::safe_to_reboot::can_reboot: 'careful'
-govuk::safe_to_reboot::reason: 'Has an encrypted partition that must be mounted manually after reboot'
+govuk::safe_to_reboot::can_reboot: 'overnight'
+govuk::safe_to_reboot::reason: 'Secondaries will reboot overnight if cluster is healthy'
```
Comment (alexmuller) [inline hieradata/class/licensify_mongo.yaml]: Licensify's mongo cluster definitely can't reboot overnight :)

Comment (mattbostock) [inline hieradata/class/licensify_mongo.yaml]: Definitely not! An overzealous search and replace there, thanks @alexmuller.

Comment (mattbostock) [inline hieradata/class/licensify_mongo.yaml]: Force-pushed to fix.

**Comment by mattbostock** [comment]
_bump_

**Comment by mattbostock** [comment]
Rebased against recent changes in master.

**Comment by surminus** [comment]
:+1: let's do this thing. merge away

**Pull request merged**